src/components/Home.js

```diff
@@ -1,8 +1,15 @@
 import React from 'react';
 
-const LandingPage = () =>
-  <div>
-    <h1>Home Page</h1>
-  </div>
+import withAuthorization from './withAuthorization';
 
-export default LandingPage;
+const HomePage = () =>
+  (
```

Comment:
To be fair, personally I would put this parenthesis in the same line as const declaration (this reduces our indendation by one level)